packages/@biomejs/js-api/README.md

```diff
@@ -26,6 +26,9 @@ const biome = await Biome.create({
 	distribution: Distribution.NODE, // Or BUNDLER / WEB depending on the distribution package you've installed
 });
 
+// Optionally apply a Biome configuration (instead of biome.json)
+biome.applyConfiguration({...});
```

Comment:
This is a correct usage in Biome v1.9. However, the current main branch tracks for Biome v2.0 (which is in beta), and it is required to open and specify a project since v2.0. Could you please add the following code to make it work with Biome v2.0@beta?

```suggestion
const projectKey = biome.openProject('path/to/project/dir');

// Optionally apply a Biome configuration (instead of biome.json)
biome.applyConfiguration(projectKey, {...});
```